site/components/pages/Pledge/PledgeDashboard/index.tsx

```diff
@@ -50,24 +50,20 @@ export const PledgeDashboard: NextPage<Props> = (props) => {
       <PageHead
         title={t({
           id: "pledges.dashboard.head.title",
-          message: "KlimaDAO | Pledges",
+          message: "`${pledgeOwnerTitle}'s Pledge | KlimaDAO`",
         })}
         mediaTitle={t({
           id: "pledges.dashboard.head.metaTitle",
-          message: `${pledgeOwnerTitle}'s pledge`,
+          message: `${pledgeOwnerTitle}'s Pledge`,
         })}
         metaDescription={t({
-          id: "shared.head.description",
-          message:
-            "Drive climate action and earn rewards with a carbon-backed digital currency.", // Do we need a better metadescription??
+          id: "pledges.dashboard.head.metaDescription",
+          message: `${pledgeOwnerTitle} pledges to Offset ${pledge.footprint} Carbon Tonnes. View their carbon offset history and read more about their commitment.`,
```

Comment:
pledge.footprint is an array, need to sum the tonnes

![image](https://user-images.githubusercontent.com/88635679/199798441-ee8469b8-1948-4980-b257-61fdba3f3473.png)

Reply:
My bad, fixed that one up.
Glad I added the total attribute to sum them all up in the db ;)

Reply:
<img width="270" alt="image" src="https://user-images.githubusercontent.com/97446324/199995375-f5c05644-b077-420c-b002-d48e32ec4450.png">
For some reason the pledged carbon footprints aren't showing, any idea on why that might be? 🤔

Reply:
I'll check out and poke around. I also notice this: 
![image](https://user-images.githubusercontent.com/88635679/200375114-a7e5825a-72b6-42b2-b7b7-6288ef90eca3.png)